ChristmasTree.java

```diff
@@ -0,0 +1,130 @@
+package LAB7.LAB7;
+
+public class ChristmasTree {
+    public int height = 0;
+    public String producer = "";
+    public float price = 0;
+    public String material = "";
+
+    private String color = "";
+    private int dateMade = 0;
+    private String width = "";
+    private String style = "";
+    private String treeFamily = "";
+
+    private static int DayWhenShouldYouPutUpYourChristmasTree = 29;
```

Comment:
even variable is static. it still should follow Java naming convention and starts from lover latter: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html